packages/manager/src/factories/types.ts

```
@@ -172,3 +172,37 @@ export const volumeTypeFactory = Factory.Sync.makeFactory<PriceType>({
   ],
   transfer: 0,
 });
+
+export const kubernetesTypeFactory = Factory.Sync.makeFactory<PriceType>({
+  // {
```

@mjac0bs @abailly-akamai 
Could one of you guys please chime in on this change?

Looking at the Tiered Pricing API Spec, the `lke/types` endpoint returns more than one object. One for `LKE Standard Availability` and the other for `LKE High Availability`. We are only concerned with the latter for removal of the hardcoded values. However, if I try to create the factory with both objects I get an error due to the shape of the `PriceType` interface.

Is it okay to omit the `LKE Standard Availability` object here for the purpose of creating the factory?

---

This ([1](https://github.com/linode/manager/pull/10468/files#diff-c4904f695ece84849bc2372518313ff64475b5598ab35b45708266fa74e2e6e8R924-R930), [2](https://github.com/linode/manager/pull/10468/files#diff-eb6a0c856c00c5d1181386d59783fe18e78288417490a40da7ffd862e9a57db5) is what I did for `object-storage/types`, which also returns multiple price objects.

---

@mjac0bs Great! I'll implement a similar pattern. Thanks for sharing.